README.md

```diff
@@ -192,19 +224,41 @@ This will exclude topics with names like "test", "test123", and "abc-testing".
 To override Jit's API endpoint, you can set the `JIT_API_ENDPOINT` environment variable. If the variable is not set, the
 default value will be used.
 
-### Settings Up Self-Hosted Runners
+## Settings Up Self-Hosted Runners
 
 To setup self-hosted runners, use the following command:
 
 You need to take the self hosted runners token from the Github Actions page of your repository.
 `https://github.com/<your-github-org-name>/jit/settings/actions/runners`
 
-#### Running on CentOS
+### Running on CentOS
+
+```shell
+make self-hosted-runner centos runner_token=<runner-token> github_org=<github-organization> jit_repo=<jit-repo-name>
```

Comment:
why do we need the jit centralize repo name? isn't that relates to self hosted runners that are configured in repo level instead of organization level?

Reply:
If so this parameter should be optional and an explanation should be added about configuring the runner to the organization or a specific repo

Reply:
removed from doc